src/main/java/space/obminyashka/items_exchange/service/impl/UserServiceImpl.java

```diff
@@ -91,6 +91,7 @@ private User mapOAuth2UserToUser(DefaultOidcUser oAuth2User) {
         final var lastName = Objects.requireNonNullElse(oAuth2User.getFamilyName(), "");
         final var password = Objects.requireNonNullElse(oAuth2User.getIdToken().getTokenValue(),
                 UUID.randomUUID().toString());
+        user.setIsOauth2Login(true);
```

Comment:
It'd be great if **loginUserWithOAuth2** method was covered with a simple unit test.